README.md

```diff
@@ -370,3 +370,11 @@ $ cargo test
 ```
 
 from the repository root.
+
+### Known issues
+
+#### I just hit Ctrl+C in the middle of ripgrep's output and now my terminal's foreground color is wrong!
+
+Type in `color` on Windows and `echo -ne "\033[0m"` on Unix to restore your original foreground color.
+
+PR [#187](https://github.com/BurntSushi/ripgrep/pull/187) fixed this, and it was later deprecated in [#281](https://github.com/BurntSushi/ripgrep/issues/281), due to an unsolvable deadlock in MSYS terminals. I full explanation is available [here](https://github.com/BurntSushi/ripgrep/issues/281#issuecomment-269093893).
```

Comment:
I don't think "unsolvable deadlock" is an accurate summary. I'd rather just say "see this PR and this issue for more details."

Also, can you please wrap this to 80 columns and find a way to shorten the header? Please use a style that's consistent with the rest of the README.